### PR TITLE
DevWs for YookSmart containing containing YookSmart Window Blinds

### PR DESCRIPTION
**Pull request by SmartThings**

### CODE REVIEW

File: devicetypes/yooksmart/yooksmart-window-blinds.src/yooksmart-window-blinds.groovy [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ *
+ *  first release for IKEA smart window blinds
+ */
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+    definition(name: "YookSmart Window Blinds", namespace: "yooksmart", author: "YookSmart", ocfDeviceType: "oic.d.blind", mnmn: "SmartThingsCommunity", vid: "58b24db0-3f12-3cd1-a86f-cc435ae6f991") {
```
Comment (KKlimczukS) [inline devicetypes/yooksmart/yooksmart-window-blinds.src/yooksmart-window-blinds.groovy:19]: Please, could You add Your devices (fingerprints) to the existing DTH that covers those functionalities ?
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

**Review by JeremeLau — APPROVED** [review]

File: devicetypes/yooksmart/yooksmart-window-blinds.src/yooksmart-window-blinds.groovy (hunk 1 above)
Comment (JeremeLau) [reply devicetypes/yooksmart/yooksmart-window-blinds.src/yooksmart-window-blinds.groovy:19]: Yes, how should I do it?

Comment (KKlimczukS) [reply devicetypes/yooksmart/yooksmart-window-blinds.src/yooksmart-window-blinds.groovy:19]: Please create a new pull request (close this one) and edit only zigbee-window-shade-battery.groovy (add Your fingerprints there. Some tweaks in the code might be needed to handle specific configurations for Your devices etc. - for example in  configure() or in parse() methods)

**Comment by k-morawski** [comment]
I am closing this pull request. Connected integration request has been failed.

**Pull request closed without merging**